src/shared/bg_public.h

```diff
@@ -144,6 +144,60 @@ struct playerState_t
 	bool IsWeaponReady() const;
 };
 
+struct entityState_t
+{
+	int          number; // entity index
+	entityType_t eType; // entityType_t
+	int          eFlags;
+
+	trajectory_t pos; // for calculating position
```

Comment:
I don't see how this could be working as the fields are in a different order from OpaqueEntityState.

Reply:
Oh there is another commit. Well, if you really want to add 2 daemon submodule references after merging in daemon, I guess that works

Reply:
Yes, I think the daemon references will unfortunately change quite a bit.